Library/Homebrew/dependencies.rbi

```diff
@@ -13,8 +13,8 @@ class Dependencies < SimpleDelegator
   sig { returns(T::Enumerator[Dependency]) }
   def each(&blk); end
 
-  sig { params(blk: T.proc.params(arg0: Dependency).returns(T::Boolean)).returns(T::Array[Dependency]) }
-  sig { returns(T::Enumerator[Dependency]) }
+  sig { override.params(blk: T.proc.params(arg0: Dependency).returns(T.anything)).returns(T::Array[Dependency]) }
```

Comment:
Just fixing a similar overly narrow return type that was fixed in `Library/Homebrew/PATH.rb` above, in an otherwise unrelated bit of code.